webdriver-spec.html

```diff
@@ -5316,106 +5316,110 @@
-      As an example:
-
-     <p><img src=graphics/note3actions.svg alt="">
-
-     <p>During tick 2, source 1 will have its pointerMove action dispatched,
-      while source 2 will do nothing.
-  </aside>
-
-  <p>
-
-  </section> <!-- /Introduction -->
+<section>
+<h2>Actions</h2>
+
+<p class=issue>Expand these examples
+
+<p>The Actions API provides a low-level interface
+ for providing virtualised device input to the web browser.
+ Conecputally, the actions API divides time into a series of <a>ticks</a>.
+ The local end sends a series of actions
+ which correspond to the change in state, if any,
+ of each input device during each tick.
+ For example, pressing a key
+ is represented by an action chain
+ consisting of a single key input device and two ticks,
+ the first containing a <a>keyDown</a> action,
+ and the second a <a>keyUp</a> action,
+ whereas a pinch-zoom input is represented
+ by an action chain consisting of a three ticks
+ and two pointer input devices of type touch,
+ each performing a sequence of actions <a>pointerDown</a>,
+ followed by <a>pointerMove</a>, and then <a>pointerUp</a>.
+
+<p class=issue>Move this note:
+
+<p class=note>An <a>activation trigger</a>
+ generated by the WebDriver API user
+ need to be indistinguishable from those generated
+ by a real user interacting with the browser.
+ In particular, the dispatched events will have
+ the <a>isTrusted</a> attribute set to true.
+ The most robust way to dispatch these events
+ is by creating them in the browser implementation itself.
+ Sending OS-specific input messages to the browser's window
+ has the disadvantage that the browser being automated
+ may not be properly isolated from a user
+ accidentally modifying input source state;
+ use of an OS-level accessibility API has the disadvantage
+ that the browser's window must be focused,
+ and as a result, multiple WebDriver instances cannot run in parallel.
+
+<aside class="example">
+ <p>Imagine we have two fingers acting on a touchscreen.
+  One finger will press down on element1 at the same moment
+  that another finger presses down on element2.
+  Once these actions are done, the first finger
+  will wait 5 seconds while the other finger moves to element3.
+  Then both fingers release from the touchscreen.
+
+ <p>When the <a>remote end</a> receives this,
+  it will look at each input source's action lists.
+  It will dispatch the first action of each source together,
+  then the second actions together, and lastly, the final actions together.
+
+ <p>The diagram below displays when each action gets executed.
+  "Source 1" is the first finger, and "source 2" is the second.
+
+ <p><img src=graphics/note1actions.svg alt="">
+
+ <p>There is no limit to the number of input sources,
+  and there is no restriction regarding the length of each input's action list.
+  Meaning, there is no requirement that all action lists have to be the same length.
```

Comment:
Nit: "This means" reads better than "meaning". If you want to use "meaning", use it in the middle of the sentence.